openfisca_aotearoa/entities.py

```diff
@@ -46,4 +46,46 @@
     is_person = True,
     )
 
-entities = [Titled_Property, Person]
+Family = build_entity(
+    key = "family",
+    plural = "families",
+    label = u'Family',
+    doc = '''
+    A Family represents a collection of related persons.
+
+    Example:
+    Family entities are required for calculation of working for families entitlements.
+
+    A family can contain a number of roles, such as 'principal_caregiver', 'partner', 'dependant_child' & 'independant_child'.
```

Comment:
These elements are readable below, and documentation can be automatically generated for them.
The documentation would add more value by providing reference to acts that define these roles  🙂